ArticleTemplates/assets/js/bootstraps/campaign.js

```diff
@@ -71,6 +72,11 @@ define([
       enableButton(form);
```

Comment:
@regiskuckaertz is this selecting a button and changing the text on it? I was attempting to copy this state but it seemed error specific..
```
       if (form.firstElementChild.className === 'campaign__error') {
         form.insertAdjacentHTML('afterbegin', '<p class="campaign__error">Sorry, there was an error submitting your contribution. Please, try again.</p>');
       }
```